modules/lang-painless/src/test/java/org/elasticsearch/painless/WhenThingsGoWrongTests.java

```diff
@@ -197,4 +201,13 @@ public void testDynamicListWrongIndex() {
             exec("def x = new ArrayList(); x.add('foo'); return x['bogus'];");
         });
     }
+
+    public void testRCurlyNotDelim() {
+        ScriptException e = expectThrows(ScriptException.class, () -> {
+            // We don't want PICKY here so we get the normal error message
+            exec("def i = 1} return 1", emptyMap(), emptyMap(), null);
+        });
```

Comment:
I'll switch it to expectScriptThrows - I had done that at first but it wasn't working with picky and I didn't notice picky was the problem until after I changed it and then didn't change it back. I'll do so now.

I turned off picky because I was trying to test that the user gets a sensible error message by default. It looks like we aren't picky by default?